mozldap.go

```diff
@@ -296,6 +296,36 @@ func (cli *Client) GetUserDNById(uid string) (dn string, err error) {
 	return
 }
 
+// GetUserUidNumber returns the UID number of a user using a shortdn
+//
+// example: cli.GetUserUidNumber("mail=jvehent@mozilla.com")
+func (cli *Client) GetUserUidNumber(shortdn string) (uidNumber string, err error) {
```

Comment:
It's a bit odd that `GetUserUidNumber` would return a string and not an int. Would it make sense to return the uidNumber as a uint64? It's pretty easy to do with
```go
uidNumber, err = strconv.Atoi(val)
if err != nil {
panic(err)
}
```